src/App.tsx

```diff
@@ -14,11 +14,66 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: ['Jam'],
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          {!selectedGood.length
+            ? 'No goods selected'
+            : `${selectedGood}
+            ${' '}
+            is selected`}
```

Comment:
For 2 or more selected goods there should be a message:
<img width="481" alt="image" src="https://user-images.githubusercontent.com/52370890/178919910-dcdbed9a-2536-411c-8663-809384b1d77b.png">

Reply:
And move this logic to a separate function, for example -`getSelectedGoodsTitle()`

Reply:
Pay attention to the comments, please add additional logic for multiple items and move it to separate function